perllib/Integrations/SCP.pm

```diff
@@ -25,11 +27,43 @@ has endpoint => (
     }
 );
 
+has log_open => (
+    is => 'ro',
+    lazy => 1,
+    builder => '_syslog_open',
+);
+
+sub _syslog_open {
+    my $self = shift;
+    my $ident = $self->config->{log_ident} or return 0;
```

Comment:
If I've understood all this correctly this will be set in the config only in places where you want to enable the logging, otherwise it'll short circuit and skip the logging?